### ISSUE TITLE
[QUESTION] Actions within Custom Groups & Layout Saves

### ISSUE THREAD

**Issue by FreshnessDND** [Larkinabout/fvtt-token-action-hud-core#197]
Creating a custom group then placing actions inside of those groups does not save the actions to the layout export. I only see the custom groups with nothing to them.

Also, logging into user profile and setting default layout to layout export is not saved for the player's browser when logging in from their setup. Its only saved for me when I log in, but not for them when they log in.

**Versions:**
- Token Action HUD Core: 1.4.18
- Token Action HUD System: 1.4.11
- System: dnd5e 2.3.1
- Foundry VTT: 11.308
- Browser: Chrome

**Comment by Dydyus** [comment]
My problem is similar, but it's with pathfinder. If I modify a group, after reloading the page, it comes back by default.
My GM has tried it, and it works for him, but it doesn't work for me or any other player. This problem has arisen since we upgraded to foundry V11.

(I tried to put a file in: Config Settings/Token Action HUD Core/Layout, but it wasn't taken into account.)

Versions:
Token Action HUD Core: 1.4.20
Token Action HUD Pathfinder 2: 1.4.15
System: Pathfinder2: 5.6.0
Foundry VTT: 11.308
Browser: Chrome

(ps : sorry for my english)
